.github/CODEOWNERS

```diff
@@ -98,14 +99,14 @@ go.sum @grafana/backend-platform
 /public/app/features/dimensions/ @grafana/grafana-edge-squad
 /public/app/features/geo/ @grafana/grafana-edge-squad
 /public/app/features/live/ @grafana/grafana-edge-squad
-/public/app/features/explore/ @grafana/observability-squad
+/public/app/features/explore/ @grafana/observability-experience-squad
 /public/app/features/plugins @grafana/plugins-platform-frontend
 /public/app/core/components/TransformersUI/spatial @grafana/grafana-edge-squad
 /public/app/plugins/panel/alertlist @grafana/alerting-squad
 /public/app/plugins/panel/barchart @grafana/grafana-bi-squad
 /public/app/plugins/panel/heatmap @grafana/grafana-bi-squad
 /public/app/plugins/panel/histogram @grafana/grafana-bi-squad
-/public/app/plugins/panel/nodeGraph @grafana/observability-squad
+/public/app/plugins/panel/nodeGraph @grafana/logs-traces-squad
```

Comment:
Could we add `/public/app/plugins/panel/logs` as well?

Reply:
done!